app/controllers/cuisine_controller.rb

```diff
@@ -0,0 +1,69 @@
+class CuisineController < ApplicationController
+
+  before_action :find_related_restaurants, :only => [:delete, :destroy]
+
+  def index
+    @cuisines = Cuisine.sorted
+  end
+
+  def show
+    @cuisine = Cuisine.find(params[:id])
```

Comment:
Since `id` is the only relevant param for this endpoint, you can actually limit it so that it has to be present:
`Cuisine.find(params.require(:id))`

Reply:
Done